src/tests/data/mac/blake2bmac.vec

```diff
@@ -0,0 +1,1051 @@
+[BLAKE2b(224)]
```

Comment:
Add a comment indicating where these came from

Reply:
The ones for 224, 256 and 384 I made manually(could not find any online not even in the RFC 😢) but did check they match outpout of Go for 256 and 384: https://play.golang.org/p/ktgwCpIfc19

Will add comments and link to source of 512 test vectors.